src/_pytest/config/__init__.py

```diff
@@ -578,12 +578,18 @@ def _set_initial_conftests(
             self._try_load_conftest(invocation_dir, importmode, rootpath)
 
     def _is_in_confcutdir(self, path: Path) -> bool:
```

Comment:
How about renaming this to focus on the purpose of the method rather than the implementation detail?

```suggestion
    def _should_load_conftests_from(self, path: Path) -> bool:
```